docs/build/dapp-development/wax-blockchain-setup/blockchain_install.md

```diff
@@ -31,33 +60,131 @@ To download the WAX Blockchain Source Code Repository:
 
 If you're using our Docker images, you do **not** need to complete these steps.
 
-To build the WAX Blockchain from source, you can use the following steps. If you have a previous version installed, you'll need to uninstall it first. Refer to [Uninstall WAX](/build/dapp-development/wax-blockchain-setup/blockchain_uninstall) for more information.
+To build the WAX Blockchain from source, you can use the following steps. If you have a previous version installed, you'll need to uninstall it first. Refer to [Uninstall WAX](https://developer.wax.io/build/dapp-development/wax-blockchain-setup/blockchain_uninstall.html) for more information.
```

Comment:
Changing the relative to absolute urls isn't an improvement, neither is adding the .html suffix, so I would recommend reverting to the original links.